source/includes/administration/_custom_ui_settings.md

```diff
@@ -0,0 +1,246 @@
+### Custom UI Settings
+
+<!-------------------- FIND CUSTOM UI SETTINGS -------------------->
+
+#### Find custom ui settings associated to an organization
+
+`GET /reseller/settings/custom_ui/find?organizationId=:id`
+
+Retrieve the custom ui settings associated to an organization. If the `organizationId` is omitted, the authenticated user's organization will be used.
+
+```shell
+# Retrieve the custom ui settings
+curl "https://cloudmc_endpoint/api/rest/reseller/settings/custom_ui/find?organizationId=10572c3d-16e5-450f-8af8-a01e50dc52d4" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": {
+    "organization": {
+      "id": "10572c3d-16e5-450f-8af8-a01e50dc52d4"
+    },
+    "id": "d785ffcb-9b03-478d-a49b-52a2ccedf1b8",
+    "version": 1,
+    "enabled": true,
+    "customUIRootUrl": "https://newui.example.com",
+    "domainToken": "example.com",
+    "applyTags": [],
+    "denyTags": [],
+    "redirectRoles": []
+  }
+}
+```
+
+Retrieve the custom UI settings associated to an organization.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The configured custom UI settings' id.
+`organization.id`<br/>*UUID* | The organization id that the custom UI settings are linked to. It cannot be changed.
+`version`<br/>*integer* | The custom UI settings version.
+`enabled`<br/>*boolean* | If the custom ui redirection is enabled or not.
+`customUIRootUrl`<br/>*string* | The base root that the user will be redirected to once the user is login.
+`domainToken`<br/>*string* | The domain name that will be used in the authentication cookie. The cookie must be acceptable by both the reseller organization custom domain and the url provided in this configuration.
+`applyTags`<br/>*Array[string]* | The list organization tags that will redirect the user to the custom ui if they match. 
+`denyTags`<br/>*Array[string]* | The list organization tags that will not redirect the user to the custom ui if they match.
+`redirectRoles`<br/>*Array[string]* | The list of roles that users will be redirected to the new UI. Possible values are `reseller`,`admin`,`user`,`guest`.
+
+
+
+<!-------------------- GET CUSTOM UI SETTINGS -------------------->
+#### Retrieve custom UI settings
+
+`GET /reseller/settings/custom_ui/:id`
+
+```shell
+# Retrieve custom UI settings
+curl "https://cloudmc_endpoint/api/rest/reseller/settings/custom_ui/d785ffcb-9b03-478d-a49b-52a2ccedf1b8" \
```

Comment:
same comment as previous for the url

Reply:
will fix all of them

Reply:
28a54025d74ba155e7339ea0a8207d38ead6aea3